.github/workflows/release.yaml

```diff
@@ -0,0 +1,27 @@
+name: Release csvcubed-models
+on:
+  release:
+    types: [published]
+
+jobs:
+  test_in_environments:
+    strategy:
+      matrix:
+        # Test in all supported versions of python in both Windows and Ubuntu.
+        python-version: ['3.9', '3.10', '3.11']
+        os: [ubuntu-latest, windows-latest]
```

Comment:
Don't we need to remove windows from this list? I thought it was only going to work under ubuntu?